pkg/internal/local/klient/kind.go

```diff
@@ -0,0 +1,128 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package klient
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/cloudoperators/greenhouse/pkg/internal/local/utils"
+)
+
+// CreateCluster - creates a kind cluster with the given name
+// if the cluster already exists, it sets the context to the existing cluster
+func CreateCluster(clusterName string) error {
+	exists, err := clusterExists(clusterName)
+	if err != nil {
+		return err
+	} else if exists {
+		utils.Logf("kind cluster with name %s already exists", clusterName)
+		return utils.Shell{
+			Cmd: "kubectl config set-context kind-${name}",
```

Comment:
I would use `NewKubeClientFromContext()` to change context

Reply:
KinD automatically sets the newly created cluster as the current-context by default. 

Since you can create cluster with `Namespace` and without `Namespace` and also execute the dev CLI actions repeatedly, this just ensures your are always in the context of your `KinD` clusters.

Regarding the method `NewKubeClientFromContext()`, initially I thought the CLI could just act on the `current-context` instead of using `kubeconfig` path or setting `KUBECONFIG` env. 

Currently the `dev` sub-command of CLI acts only on `KinD` clusters, so we rely on the `KinD` cluster name and not on the context. So `NewKubeClientFromContext()` is an unused function and I will remove it.